client/state/analytics/middleware.js

```diff
@@ -33,7 +33,7 @@ const eventServices = {
 
 const pageViewServices = {
 	ga: ( { url, title } ) => analytics.ga.recordPageView( url, title ),
-	default: ( { url, title } ) => analytics.pageView.record( url, title ),
+	default: ( { url, title, ...params } ) => analytics.pageView.record( url, title, params ),
```

Comment:
This change now makes all payload properties to be injected in the Tracks event, which is probably something we want to avoid. For example, an `undefined` `service` property is now attached to pageviews:

<img width="382" alt="screenshot" src="https://user-images.githubusercontent.com/594356/33938661-99bc4bfa-e008-11e7-8885-60a9451c78e8.png">